board_client/animations/utils.py

```
@@ -83,6 +90,182 @@ def falloff(dy, width):
+    """Generic object pool for any type with reset() method"""
+    def __init__(self, factory_func, initial_size=10, max_size=50):
+        self.factory_func = factory_func
+        self.available = [factory_func() for _ in range(initial_size)]
+        self.in_use = []
+        self.max_size = max_size
+    
+    def acquire(self):
+        if self.available:
+            obj = self.available.pop()
+        else:
+            obj = self.factory_func()
```

Ruff (0.12.2)

Use ternary operator `obj = self.available.pop() if self.available else self.factory_func()` instead of `if`-`else`-block

Replace `if`-`else`-block with `obj = self.available.pop() if self.available else self.factory_func()`

(SIM108)